test/acsengine.deploy

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env groovy
+
+node {
+  withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: 'AZURE_CLI_SPN_ACS_TEST',
```

Comment:
Might want to re-use the existing SP credentials ID rather than adding a new one. If not, we should cleanup the old one when we remove the old job.